src/app.js

```diff
@@ -1 +1,186 @@
-console.log('Hallo daar!');
\ No newline at end of file
+console.log('Hallo daar!');
+import axios from 'axios';
+
+const countryList = document.getElementById("country-List");
+const URI = "https://restcountries.com/v2/";
+const ENDPOINT = "all";
+
+async function fetchCountryInfo() {
+    try {
+        const response = await axios.get(URI + ENDPOINT);
+        //sorteren van de array op populatie
+        response.data.sort((a, b) => a.population - b.population)
+        console.log(response.data);
+        //laat de naam van 1 land zien,  bijv van tweede land uit de array
+        // const itemName = document.createElement("li");
+        // const itemPopulation = document.createElement("article");
+        // const itemFlag = document.createElement("img");
+        // itemName.textContent = response.data[1].name;
+        // itemPopulation.textContent = `has a population of ${response.data[1].population} people`;
+        // itemFlag.textContent = response.data[1].flags;
+        // item.textContent = response.data[1].population;
+        response.data.map((country) => {
```

Comment:
Je zou hier nog spreading kunnen toepassen, zodat je niet overal `country.` voor hoeft te zetten
```suggestion
        response.data.map(({ flag, region, name, population }) => {
```